src/components/learner-credit-management/AssignmentTableRemind.jsx

```diff
@@ -35,29 +50,82 @@ const AssignmentTableRemindAction = ({
     close,
     isOpen,
     open,
-  } = useRemindContentAssignments(assignmentConfigurationUuid, assignmentUuids, shouldRemindAll);
+  } = useRemindContentAssignments(assignmentConfiguration.uuid, assignmentUuids, shouldRemindAll);
 
   const selectedRemindableRowCount = calculateTotalToRemind({
     assignmentUuids,
     isEntireTableSelected: shouldRemindAll,
     learnerStateCounts,
   });
 
+  const {
+    BUDGET_DETAILS_ASSIGNED_DATATABLE_OPEN_BULK_REMIND_MODAL,
+    BUDGET_DETAILS_ASSIGNED_DATATABLE_CLOSE_BULK_REMIND_MODAL,
+    BUDGET_DETAILS_ASSIGNED_DATATABLE_BULK_REMIND,
+  } = EVENT_NAMES.LEARNER_CREDIT_MANAGEMENT;
+
+  const trackEvent = (eventName) => {
+    // constructs a learner state object for the select all state to match format of select all on page metadata
+    const learnerStateObject = {};
+    learnerStateCounts.forEach((learnerState) => {
+      learnerStateObject[learnerState.learnerState] = learnerState.count;
+    });
+
+    const selectedRowsMetadata = isEntireTableSelected
+      ? { uniqueLearnerState: learnerStateObject, totalSelectedRows: tableInstance.itemCount }
```

Comment:
[clarification] In the event of the entire table being selected for reminders, not all table rows are remindable (unlike cancelations, where all rows are cancelable). Is the `totalSelectedRows` misleading in this case given that we would be logging the total number of rows in the table versus the number of remindable rows?

Reply:
Went with `selectedRemindedRowCount` to populate the `totalSelectedRows` object. 👍🏽